core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala

```diff
@@ -34,6 +34,8 @@ import org.apache.spark.util.{ShutdownHookManager, Utils}
  */
 private[spark] class DiskBlockManager(conf: SparkConf, deleteFilesOnStop: Boolean) extends Logging {
 
+  private val METADATA_FILE_SUFFIX = ".meta"
```

Comment:
Hmm, good point... there's currently no metadata kept in the `DiskStore` class, but then this shouldn't be a lot of data.